src/Main.java

```diff
@@ -6,5 +6,101 @@ public static void main(String[] args) {
         // to see how IntelliJ IDEA suggests fixing it.
         System.out.println("Hello and welcome!");
         System.out.println("variable-Lesson2");
+        System.out.println("Задача 1");
+        int A = 4000;
+        System.out.println("Значение переменной A " + "с типом int равно " + A);
+        byte B =-120;
+        System.out.println("Значение переменной B " + "с типом byte равно " + B);
+        short D = 30500;
+        System.out.println("Значение переменной D " + "с типом short равно " + D);
+        long E =-7225327136854600200L;
+        System.out.println("Значение переменной E " + "с типом long равно " + E);
+        float F = 3.568F;
```

Comment:
для float должна используется маленькая буква f, для лонга L
```suggestion
        float F = 3.568f;
```